### PR TITLE
238 - Move UI config out of data layer

### LINKED ISSUE #238: UI classes need to be removed from data layer
For now `hard_data.dart` and the classes within it are our data layer so UI classes like `ReusableCard` have no business being there and needed to be refactored into separate datamodels holding the data, which is displayed by the `ReusabelCard` widgets

### PR DESCRIPTION
**Pull request by maks**
Refactor that removes all UI configuration from the data layer (which is currently just hardcoded in `hard_data.dart`)

## Screenshots

There should be no visual changes to the existing app as the refactor just cleans up the app code architecture.

Fixes: #238

### CODE REVIEW

**Review by workerbee22 — APPROVED** [review]
LGTM

File: lib/hard_data.dart [hunk 1]
```diff
@@ -1,17 +1,15 @@
-// A list of two cards
-import 'package:flutter/material.dart';
+// import 'package:flutter/material.dart';
```
Comment (domesticmouse) [inline lib/hard_data.dart:1]: nit: prefer deleting outright to commenting out. We use source control for a reason =)

**Review by domesticmouse — COMMENTED** [review]
LGTM

File: lib/hard_data.dart (hunk 1 above)
Comment (maks) [reply lib/hard_data.dart:1]: Doh! thanks for spotting that @domesticmouse , yes that's a plain mistake I'm the same - no leaving commented out code!
Unfort this has been merged, I'll make sure to fix that in my next PR !

**Pull request merged**